packages/documentation/copy/en/reference/JSX.md

```diff
@@ -452,7 +452,7 @@ By default, function components must return `JSX.Element | null`. However, this
 namespace JSX {
     export type ElementType =
         // All the valid lowercase tags
-        keyof IntrinsicAttributes
+        keyof IntrinsicElements
```

Comment:
I think this is correct, but, this code's syntax is definitely already wrong; this should be:

```ts
export type ElementType =
    | keyof IntrinsicElements
    | ((props: any) => Element)
    | (new (props: any) => ElementClass)
    ;
```

Or something.

Reply:
oh right, I missed that